packages/runtime/package.json

```diff
@@ -116,8 +116,8 @@
     "@emotion/server": "^11.4.0",
     "@emotion/sheet": "^1.0.3",
     "@emotion/utils": "^1.0.0",
-    "@makeswift/next-plugin": "0.3.0",
-    "@makeswift/prop-controllers": "^0.1.0",
+    "@makeswift/next-plugin": "workspace:*",
+    "@makeswift/prop-controllers": "workspace:*",
```

Comment:
Switch to the [`workspace:` protocol](https://pnpm.io/workspaces#workspace-protocol-workspace); the non-workspace references were observed to be pulled from the registry instead of the repo in some cases, even when the local package version and the dependency version range matched exactly.